src/main2.js

```diff
@@ -0,0 +1,16 @@
+import http from 'http';
+import socketio from 'socket.io';
+import Monopoly from "./monopoly";
+import Property from "./property";
+import fs from 'fs';
+//import "./cardProps/properties.json";
+let PropertyArray = new Array(0);
```

Comment:
These should probably be const.  You're not going to be reassigning them at any point.

const doesn't prevent you from changing an object's contents, just prevents you from assigning something new to a variable.